deps/build.jl

```diff
@@ -58,8 +63,7 @@ provides(AptGet,
         "libpango1.0-0" => [pango,pangocairo],
         "libglib2.0-0" => gobject,
         "libpng12-0" => libpng,
-        "libpixman-1-0" => pixman,
-        "gettext" => gettext
```

Comment:
> [gettext is] actually needed by GLib

I don't know the details, but I suspect that it depends on how GLib is built. Amazon Linux has GLib but not gettext.